### PR TITLE
Corrected StackOverflow question tags

### PR DESCRIPTION
**Pull request by ry8806**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ A description of all the repos is [here](https://github.com/aspnet/Home/wiki/Rep
 Our team members also monitor several other discussion forums:
 
 * [ASP.NET vNext forum](http://forums.asp.net/1255.aspx/1?ASP+NET+vNext)
```
Comment (pranavkm) [inline CONTRIBUTING.md]: Could you also change this while you're at it:

> \[ASP.NET Core forum\]\(http://forums.asp.net/1255.aspx/1?ASP+NET+Core)

**Comment by ry8806** [comment]
@pranavkm done :)

**Comment by pranavkm** [comment]
Didn't notice the Jabbr link until now. I don't think anybody is on Jabbr these days. I'll have this removed as part of merging it in. 
@Eilon, does this look good to you?

**Comment by Eilon** [comment]
:shipit:

**Comment by pranavkm** [comment]
Sorry, I just noticed that you were working off the master branch which doesn't look like it's being maintained. The README in the dev branch is vastly different - https://github.com/aspnet/Home and doesn't have any of the stackoverflow references or the Asp.Net forum references.

@danroth27 could we bring the master branch up to speed (or remove it)?

**Comment by Eilon** [comment]
Oh yeah I was surprised that we still had such old stuff... then I think really there's no action here. Not sure how someone would find the master branch, seeing as how it isn't the default. Maybe some old link?

**Pull request closed without merging**